### PR TITLE
only show delegate on Usul, and hide send assets completely

### PR DESCRIPTION
**Pull request by herbig**
## Description

- only shows Delegate for Usul DAOs
- hides Send Assets permanently, for now pending transaction work needed

## Issue / Notion doc (if applicable)

closes https://github.com/decent-dao/fractal-interface/issues/728

### CODE REVIEW

File: src/pages/Treasury/index.tsx [hunk 1]
```diff
@@ -14,7 +14,9 @@ function Treasury() {
     treasury: { assetsFungible },
   } = useFractal();
   const { t } = useTranslation('treasury');
-  const showButton = assetsFungible.length > 0; // TODO also only show if the user is a DAO signer or delegated token holder..
+  // TODO show only if the user is a DAO signer or delegated token holder and assets > 0
```
Comment (Da-Colon) [inline src/pages/Treasury/index.tsx:17]: I originally had a spot in governance for 'owner'. but with the `account` object, might be good to load that data there once the DAO has been loaded

File: src/pages/Proposals/index.tsx [hunk 2]
```diff
@@ -6,21 +6,24 @@ import PageHeader from '../../components/ui/Header/PageHeader';
...
 
 export function Governance() {
   const { t } = useTranslation(['common', 'proposal']);
   const {
     gnosis: { daoName },
+    governance,
   } = useFractal();
-
+  const delegate = useFractalModal(ModalType.DELEGATE);
+  const isUsul = governance.type === GovernanceTypes.GNOSIS_SAFE_USUL;
```
Comment (Da-Colon) [inline src/pages/Proposals/index.tsx:18]: I thought about adding a boolean on the object just to hold this info for quick access?

**Review by Da-Colon — APPROVED** [review]
LGTM

File: src/pages/Proposals/index.tsx (hunk 2 above)
Comment (herbig) [reply src/pages/Proposals/index.tsx:18]: Yeah I like that idea.

**Pull request merged**